nilearn/connectome/group_sparse_cov.py

```diff
@@ -319,53 +307,32 @@ def _group_sparse_covariance(
     alpha2 = alpha**2
 
     for n in range(max_iter):
-        if max_norm is not None:
-            suffix = f" variation (max norm): {max_norm:.3e} "
-        else:
-            suffix = ""
-        if verbose > 1:
```

Comment:
this if block can be removed as the log function will directly handle the verbose level